src/main/java/org/kumoricon/registration/model/attendee/AttendeeService.java

```diff
@@ -95,6 +97,14 @@ void revokeAttendeeMembership(Attendee attendee, User user) {
         }
     }
 
+    @Transactional
+    public void revertAttendeeCheckin(Attendee attendee, User user) {
+        attendee.setCheckedIn(false);
```

Comment:
Techinically there's a race condition here - it's reading an Attendee record in one transaction but then saving it in a different one. Granted, odds are slim. Reading the Attendee record inside revertAttendeeCheckin would take care of it (because it's in a SQL Transaction)